srcpkgs/jupyterlab/template

```diff
@@ -17,4 +17,8 @@ make_check=no
 
 post_install() {
 	vlicense LICENSE
+	vmkdir etc/profile.d
+	cat > ${DESTDIR}/etc/profile.d/jupyterlab.sh <<EOF
+export JUPYTERLAB_DIR=$HOME/.local/share/jupyter/lab
+EOF
```

Comment:
Even though this is a one-line script, let's put it in `files/jupyterlab.sh` so you can just
```sh
vinstall "${FILESDIR}/jupyterlab.sh" 644 etc/profile.d
```
here. It's nice to keep things out of heredocs.

Reply:
Done.